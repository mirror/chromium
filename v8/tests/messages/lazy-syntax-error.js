// Test that we can parse the whole file without issuing syntax errors
// and run it until throwing an error at the end.  The exception is
// thrown because message tests must terminate with a non-zero error
// code.

function testIncrementCall() {
  ++fun();
}

print("all parsed");

throw "error";
