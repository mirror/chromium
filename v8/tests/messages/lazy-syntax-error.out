all parsed
lazy-syntax-error.js:12: Uncaught error
throw "error";
^
